### DISCUSSION TITLE
iOS 16 Lock Screen widget support

### DISCUSSION THREAD

**Discussion by neponn** [home-assistant/frontend#13821] [Other feature requests] [answered]
Hi,

I feel bad asking, because I know there are a heap of iOS 16 related bugs to work through, but when the time is right, it would be great to add the ability to create widgets that can be displayed on the iOS 16 Lock Screen. iOS ships with some cool weather / activity widgets, but it would be great if we can add HA derived sensor widgets as we can with the watch.

![weather-lock-screen](https://user-images.githubusercontent.com/32885870/191194929-b573b4af-3843-49a2-8208-1731eab9d1f1.jpg)

**Comment by frenck** [comment, marked as answer]
This is not frontend related request.

Please open up an feature request here: <https://community.home-assistant.io/c/feature-requests/13>

Thanks 👍
